src/Gaufrette/Adapter/Local.php

```diff
@@ -309,4 +315,29 @@ protected function createDirectory($directory)
             throw new \RuntimeException(sprintf('The directory \'%s\' could not be created.', $directory));
         }
     }
+
+    public static function deleteDirectory($directory)
```

Comment:
This method should be private and tests should use the filesystem to delete their base directory.

Reply:
Sorry but i can't remove static. In tests i can't use the method `$this->filesystem->delete($this->directory)` because the filesystem checks file with this method `assertHasFile`and not working with directory.